{% extends 'base.html.twig' %}

{% block title %}Hello !{% endblock %}

{% block body %}
<style>
    .example-wrapper { margin: 1em auto; max-width: 800px; width: 95%; font: 18px/1.5 sans-serif; }
    .example-wrapper code { background: #F5F5F5; padding: 2px 6px; }
</style>

<div class="example-wrapper">

    This friendly message is coming from:
    <ul>
        <li>Your controller at <code>src/Controller/UserController.php</code></li>
        <li>Your template at <code>templates/user/index.html.twig</code></li>
    </ul>
    {{ form_start(form) }}
    {# render the task's only field: description #}
    {{ form_row(form.description) }}

    <h3>Tags</h3>
    {#<ul class="tags">#}
        {# iterate over each existing tag and render its only field: name #}
        {#{% for tag in form.tags %}#}
            {#<li>{{ form_row(tag.name) }}</li>#}
        {#{% endfor %}#}
    {#</ul>#}
    <ul class="tags" data-prototype="{{ form_widget(form.tags.vars.prototype)|e('html_attr') }}">
        ...
    </ul>
    {{ form_end(form) }}
</div>
{% endblock %}
